### ISSUE TITLE
Invert interface issue

### ISSUE THREAD

**Issue by ZacSpec** [DarkMachines/high-dimensional-sampling#30]
When using the invert interface on the simple function, the invert applies to the original function as well, which led to the pyscannerbit interface reporting the negative of the actual function values. 

```
simple = function.get_simple_interface_with_scan()
simple.invert(True)
...
y = function(x)
```

Fixed by returning `-function(x)` instead.

**Comment by ZacSpec** [comment]
Issue as requested @bstienen , sorry about delay

**Comment by bstienen** [comment]
Thanks for creating the issue! I have solved this in pull request #31, which is now included in the package's code.
